client/state/global-sidebar/selectors.ts

```diff
@@ -84,15 +83,8 @@ export const getShouldShowCollapsedGlobalSidebar = (
 		isScheduledUpdatesMultisiteCreateRoute( state ) ||
 		isScheduledUpdatesMultisiteEditRoute( state );
 
-	const isBulkDomainsDashboard = isInRoute( state, [ '/domains/manage' ] );
-	const isSmallScreenDashboard =
-		( isSitesDashboard || isBulkDomainsDashboard ) && isWithinBreakpoint( '<782px' );
-
 	return (
-		isSiteJustSelectedFromSitesDashboard ||
-		isSiteDashboard ||
-		isPluginsScheduledUpdatesEditMode ||
-		isSmallScreenDashboard
```

Comment:
this smallScreenDashboard collapsed menu no longer makes sense with the unified Masterbar and sidebar toggling behavior. Lets keep things consistent and just toggle a full width sidebar at these breakpoints as we do everywhere else.